detections/web/ws_ftp_remote_code_execution.yml

```diff
@@ -0,0 +1,62 @@
+name: WS FTP Remote Code Execution
+id: b84e8f39-4e7b-4d4f-9e7c-fcd29a227845
+version: 1
+date: '2023-10-01'
+author: Michael Haag, Splunk
+status: production
+type: TTP
+data_source: []
+description: 'The following analytic is designed to detect a Remote Code Execution (RCE) vulnerability (CVE-2023-40044) in WS_FTP, a managed file transfer software by Progress. The search specifically looks for HTTP requests to the "/AHT/AhtApiService.asmx/AuthUser" URL with a status of 200, which could indicate an exploitation attempt.'
+search: '| tstats count min(_time) as firstTime max(_time)
+  as lastTime from datamodel=Web where Web.url IN ("/AHT/AhtApiService.asmx/AuthUser") Web.status=200
```

Comment:
Mike, should we specifically look for `http_method = POST` ?

https://www.assetnote.io/resources/research/rce-in-progress-ws-ftp-ad-hoc-via-iis-http-modules-cve-2023-40044

Reply:
I will add it to the detection